frontend/src/App.tsx

```diff
@@ -17,6 +17,41 @@ function App(): ReactElement {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   const global = useContext(store) as any;
   const state = global.state;
+  const dispatch = global.dispatch;
+
+  useEffect(() => {
```

Comment:
Since this is in the main app file, this should be only called on the initial render on the app. It might be a good idea to add a check for if it is the first load to make sure, but I wouldn't consider it necessary.